content/ja/docs/reference/glossary/horizontal-pod-autoscaler.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Horizontal Pod Autoscaler
+id: horizontal-pod-autoscaler
+date: 2018-04-12
+full_link: /ja/docs/tasks/run-application/horizontal-pod-autoscale/
+short_description: >
+  水平Pod自動スケーラーは、ターゲットCPU使用率またはカスタムメトリクスターゲットに基づいて、Podのレプリカ数をスケーリングするAPIリソースです。
+
+aka: 
+- HPA
+tags:
+- operation
+---
+水平Pod自動スケーラーは、ターゲットCPU使用率またはカスタムメトリクスターゲットに基づいて{{< glossary_tooltip text="Pod" term_id="pod" >}}のレプリカ数をスケーリングするAPIリソースです。
+
+<!--more--> 
+HPAは通常{{< glossary_tooltip text="ReplicationControllers" term_id="replication-controller" >}}、{{< glossary_tooltip text="Deployments" term_id="deployment" >}}または{{< glossary_tooltip text="ReplicaSets" term_id="replica-set" >}}で使用されます。HPAは{{< glossary_tooltip text="DaemonSets" term_id="daemonset" >}}などのスケーリングをサポートしないオブジェクトでは使用できません。
```

Comment:
```suggestion
HPAは通常{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}、{{< glossary_tooltip text="Deployment" term_id="deployment" >}}または{{< glossary_tooltip text="ReplicaSet" term_id="replica-set" >}}で使用されます。HPAは{{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}などのスケーリングをサポートしないオブジェクトでは使用できません。
```

We use the singular, not the plural form

https://kubernetes.io/ja/docs/contribute/localization/#terminology

> またこれらの単語は、複数形ではなく単数形を用います。 例えば、原文に"pods"と表記されている場合でも、日本語訳では"Pod"と表記してください。